website/pk-web/dirtrees.go

```diff
@@ -125,7 +129,12 @@ func (b *treeBuilder) newDirTree(fset *token.FileSet, path, name string, depth i
 	if ndirs > 0 {
 		dirs = make([]*Directory, ndirs)
 		i := 0
-		for _, d := range list {
+		for _, entry := range list {
```

Comment:
It would be nicer to convert the entry only once

Reply:
done